timesketch/lib/py2to3.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""The Python 2 and 3 compatible type definitions.
+
+File copied from the plaso project.
+"""
+import sys
```

Comment:
Don't use this, use six instead, it has all the same functionality.

Reply:
ack.... much better to just use six... done that